src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,23 @@
-export const CommentList = () => <>Put the list here</>;
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+
+export const CommentList = ({ comments }) => {
+  // if there are no comments - returning a "No comments yet" message
+  if (comments.length <= 0) {
```

Comment:
[Not critical]
Array length can never be less than zero, so this conditional can be simplified: `comments.length === 0`